lib/requirements/topic_checker.rb

```diff
@@ -12,7 +12,7 @@ def pre_publish_issues(rescue_api_errors: true)
       issues = CheckerIssues.new
 
       begin
-        if document.document_type.topics && document.topics.none?
+        if document.editions.last.document_type.topics && document.topics.none?
```

Comment:
Probably want to use `current_edition` here, so `document.current_edition.document_type.topics`.

Reply:
did not know this was a thing. Nice one, cheers

Reply:
Looks like this has slipped back into commit fa92eac41c9bc1b8fc8edfee7b59f2d1c65c95ba

Reply:
Have removed this for a 2nd time!